internal/conditions/conditions.go

```diff
@@ -18,34 +21,34 @@ const (
 )
 
 const (
-	ReasonNoPipelineDeployed           = "NoPipelineDeployed"
-	ReasonReferencedSecretMissing      = "ReferencedSecretMissing"
-	ReasonMaxPipelinesExceeded         = "MaxPipelinesExceeded"
-	ReasonResourceBlocksDeletion       = "ResourceBlocksDeletion"
-	ReasonConfigurationGenerated       = "ConfigurationGenerated"
-	ReasonDeploymentNotReady           = "DeploymentNotReady"
-	ReasonDeploymentReady              = "DeploymentReady"
-	ReasonDaemonSetNotReady            = "DaemonSetNotReady"
-	ReasonDaemonSetReady               = "DaemonSetReady"
-	ReasonAllDataDropped               = "AllTelemetryDataDropped"
-	ReasonSomeDataDropped              = "SomeTelemetryDataDropped"
-	ReasonBufferFillingUp              = "BufferFillingUp"
-	ReasonGatewayThrottling            = "GatewayThrottling"
-	ReasonNoLogsDelivered              = "NoLogsDelivered"
-	ReasonFlowHealthy                  = "Healthy"
-	ReasonTLSCertificateInvalid        = "TLSCertificateInvalid"
-	ReasonTLSPrivateKeyInvalid         = "TLSPrivateKeyInvalid"
-	ReasonTLSCertificateExpired        = "TLSCertificateExpired"
-	ReasonTLSCertificateAboutToExpire  = "TLSCertificateAboutToExpire"
-	ReasonTLSCertificateKeyPairInvalid = "TLSCertificateKeyPairInvalid"
-
-	ReasonMetricAgentNotRequired  = "AgentNotRequired"
-	ReasonMetricComponentsRunning = "MetricComponentsRunning"
-
+	// Universal pipeline reasons
+	ReasonNoPipelineDeployed      = "NoPipelineDeployed"
+	ReasonComponentsRunning       = "ComponentsRunning"
+	ReasonReferencedSecretMissing = "ReferencedSecretMissing"
```

Comment:
ReferencedSecretMissing is a common pipeline reason, the rest are Telemetry only reasons (not universal)